jiant/proj/main/write_task_configs.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+import jiant.utils.python.filesystem as py_filesystem
+import jiant.utils.python.io as py_io
+import jiant.utils.zconf as zconf
+
+
+def get_task_config(task_config_templates, task_name, task_data_dir):
+    task_config = task_config_templates[task_name].copy()
+    task_config["paths"] = {}
+    # TODO: Handle nested paths dict
```

Comment:
can this TODO be removed? or can it be handled now? not sure what it refers to

Reply:
I'll shift this to the other PR, but will add a little more detail.